provider/designate_test.go

```diff
@@ -114,7 +114,7 @@ func (c fakeDesignateClient) UpdateRecordSet(zoneID, recordSetID string, opts re
 	if opts.Description != nil {
 		rs.Description = *opts.Description
 	}
-	rs.TTL = opts.TTL
```

Comment:
this isn't currently configurable anywhere, on the new version it requires a pointer which if empty was causing a crash, seems to be fine if it's not set.

Reply:
For testing it would be ok to omit it.